backend/controllers/movieController.js

```diff
@@ -27,8 +29,44 @@ const getMovieEntry = async (req, res, next) => {
   } catch (err) { next(err); }
 };
 
+const addComment = async (req, res, next) => {
+  try {
+    const newComment = {
+      userId: req.body.userId,
+      comment: req.body.comment,
+    };
+    const imdbCode = req.params.imdb_code;
+
+    if (newComment.comment === '') throw detailedError();
```

Comment:
If I forget to include _comment_ property in the body, will it create an empty comment?

Reply:
good spot! It does indeed create an empty comment filed without the property. Will fix all the issues tomorrow, thanks again for taking the time to review it!